{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--f283cd8b-467a-45ed-aaed-3875fb883dc9",
    "created": "2024-08-13T16:15:24.061134Z",
    "modified": "2024-08-13T16:15:24.061138Z",
    "name": "No Title Available",
    "description": "IBM Robotic Process Automation with Automation Anywhere 11 could allow an attacker with specialized access to obtain highly sensitive from the credential vault. IBM X-Force ID: 160758.",
    "external_references": [
        {
            "source_name": "cve",
            "external_id": "CVE-2019-4295"
        }
    ],
    "metrics": [
        {
            "cvssV3_0": {
                "attackComplexity": "LOW",
                "attackVector": "NETWORK",
                "availabilityImpact": "NONE",
                "baseScore": 4.9,
                "baseSeverity": "MEDIUM",
                "confidentialityImpact": "HIGH",
                "exploitCodeMaturity": "UNPROVEN",
                "integrityImpact": "NONE",
                "privilegesRequired": "HIGH",
                "remediationLevel": "OFFICIAL_FIX",
                "reportConfidence": "CONFIRMED",
                "scope": "UNCHANGED",
                "temporalScore": 4.3,
                "temporalSeverity": "MEDIUM",
                "userInteraction": "NONE",
                "vectorString": "CVSS:3.0/A:N/S:U/AV:N/AC:L/C:H/PR:H/UI:N/I:N/RC:C/E:U/RL:O",
                "version": "3.0"
            }
        }
    ],
    "references": [
        {
            "tags": [
                "x_refsource_CONFIRM"
            ],
            "url": "http://www.ibm.com/support/docview.wss?uid=ibm10884840"
        },
        {
            "name": "ibm-rpa-cve20194295-info-disc (160758)",
            "tags": [
                "vdb-entry",
                "x_refsource_XF"
            ],
            "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/160758"
        }
    ]
}